packages/cli/lib/services/migration.service.ts

```diff
@@ -65,7 +65,7 @@ async function moveFile(source: string, destination: string, debug = false): Pro
 async function updateModelImport(filePath: string, debug = false): Promise<void> {
     try {
         const data = await fs.promises.readFile(filePath, 'utf8');
-        const updatedData = data.replace(/from '\.\/models/, "from '../../models");
+        const updatedData = data.replace(/from (["'])\.\/models/g, 'from $1../../models');
```

Comment:
why not simply `data.replace(/\.\/models/g, '../../models');` so we don't have to worry about what kind of quotes are being used?

Reply:
Good call, updated, thanks!